cmd/grumble/server.go

```diff
@@ -1344,7 +1352,17 @@ func (server *Server) Port() int {
 	return port
 }
 
-// Returns the port the server is currently listning
+// Returns the port the web server will listen on when it is
+// started. Returns 0 on failure.
```

Comment:
> Returns 0 on failure.
Doesn't seem like it?

Reply:
Line 1338 says it too (I just implemented the web port retrieval in exactly the same way). I'll get rid of the comment for both.